### PR TITLE
ci: fix code ownership for size tracking

### PR DESCRIPTION
**Pull request by kara**
It should be set up so either Igor OR Kara can approve
size tracking rather than both.

See https://github.com/angular/angular/pull/34043 for example

### CODE REVIEW

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
the ci is failing due to payload size issues?

**Comment by IgorMinar** [comment]
the PR maybe needs a rebase?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -347,6 +347,12 @@
 #   - IgorMinar
 #   - josephperrott
 
+# ===========================================================
+#  @angular/fw-size-tracking
+# ===========================================================
+#
+#   - kara
+#   - IgorMinar
```
Comment (gkalpak) [inline .github/CODEOWNERS]: We use alphabetic order in all other teams 😇

**Comment by gkalpak** [comment]
I pushed a fixup commit (to restore alphabetic order and consistent linebreaks 😇) and I also rebased on master, while I was at it.

**Comment by kara** [comment]
@gkalpak Thanks for fixing it up!!

**Pull request closed without merging**